src/ctia/import/capec.clj

```diff
@@ -0,0 +1,164 @@
+(ns ctia.import.capec
+  (:require [clojure.xml :as xml]
+            [ctim.schemas.ttp :as ttp]
+            [schema.core :as s]
+            [ctia.lib.time :as time]
+            [clojure.pprint :refer [pprint]]
+            [ctia.import.threatgrid.http :as http]))
+
+(defn capec-tag [kw]
+  (keyword (str "capec" kw)))
+
+(defn sieve
+  "Helps keep clojure.xml traversal code more concise, with sensible defaults."
+  ([e tag] (sieve e tag first :content))
+  ([e tag scope] (sieve e tag scope :content))
+  ([e tag scope child] (->> e
+                            (filter #(= (:tag %) (capec-tag tag)))
+                            scope
+                            child)))
+
+(defn attack-pattern-elements [doc]
+  (-> doc
+      :content
+      (sieve :Attack_Patterns)
+      (sieve :Attack_Pattern seq seq)))
+
+(s/defschema ExploitTargetSummary
+  {(s/required-key "exploit_target_id") s/Str})
+
+(s/defschema ResourcesSummary
+  {(s/required-key "tools") [{:description (s/maybe s/Str)}]})
+
+(s/defschema AttackPatternSummary
+  {(s/required-key "title") s/Str
+   (s/required-key "description") s/Str
+   (s/required-key "capec_id") s/Str
+   (s/required-key "exploit_targets") [ExploitTargetSummary]
+   (s/required-key "resources") ResourcesSummary
+   (s/required-key "indicators") [(s/maybe s/Str)]})
```

Comment:
You can use `(schema-tools.core/required-keys {"title" s/Str "description" s/Str ...` here.